src/error.rs

```diff
@@ -200,24 +241,26 @@ impl Error {
   }
 
   pub(crate) fn print_backtrace(&self, write_color: &mut impl WriteColor) {
-    if let Some(backtrace) = ErrorCompat::backtrace(self) {
-      BacktracePrinter::new()
-        .add_frame_filter(Box::new(|frames| {
-          frames.retain(
-            |frame| match frame.filename.as_ref().and_then(|x| x.to_str()) {
-              Some(file) => {
-                !(file.starts_with("/rustc/")
-                  || file.contains(&format!(
-                    "{}.cargo{}registry{}",
-                    MAIN_SEPARATOR, MAIN_SEPARATOR, MAIN_SEPARATOR
-                  )))
-              }
-              None => false,
-            },
-          );
-        }))
-        .print_trace(backtrace, write_color)
-        .ok();
+    if cfg!(not(test)) {
```

Comment:
The diffs looks big, but it's just this added `if` wrapped around the function body. Nothing else changed.

Reply:
Reverted this, since this is not needed with the new `Backtrace` wrapper.